auth/settings/base.py

```diff
@@ -114,3 +115,17 @@
 
 BRAINTREE_PUBLIC_KEY = os.environ.get('BRAINTREE_PUBLIC_KEY')
 BRAINTREE_PRIVATE_KEY = os.environ.get('BRAINTREE_PRIVATE_KEY')
+
+# Zippy config to a local docker instance.
+url = os.environ.get('ZIPPY_BASE_URL', 'http://zippy:2605/')
+ZIPPY_CONFIGURATION = {
+    'reference': {
+        # No trailing slash.
```

Comment:
not sure what this comment is about. The default URL you gave it has a trailing slash but maybe it doesn't matter?